Week3/homework/step2-2.js

```diff
@@ -2,19 +2,29 @@
 
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
-
-  // Replace this comment and the next line with your code
+  for (let i = startIndex; i < stopIndex + 1; i++) {
+    numbers.push(i);
+  }
+  for (const value of numbers) {
+    if (value % 3 === 0) {
+      threeCallback(value);
+      if (value % 5 === 0) {
+        fiveCallback(value);
+      }
+    }
+    if (value % 5 === 0 && !(value % 3 === 0)) {
+      fiveCallback(value);
+    }
+  }
```

Comment:
This logic seems a bit convoluted. You need to call `threeCallBack` if the number is divisible by 3 and `fiveCallBack` if divisible by 5. You only need two `if` statements like this:

```js
for (const value of numbers) {
  if (value % 3 === 0) {
    threeCallback(value);
  }
  if (value % 5 === 0) {
    fiveCallback(value);
  }
}
```

Reply:
I don't know why but somehow I thought only one of if statements code will work. but no, if statements run one after another.
I updated my code